### PR TITLE
update i18n scripts for syncing code.org and move them to bin

### PR DESCRIPTION
**Pull request by tanyaparker**
**Goal**: make it easier for someone else to run these scripts so I'm not the only one who can run them. Now, the only thing an engineer would need is the crowdin.yaml file from me and they could run all the scripts.

**Done**:
- [x] Move code.org syncing scripts to bin, so i18n folder just contains the translations.
- [x] Combine all the ruby scripts into one. They don't need to be run separately, and it's easier to know how to run everything from one place. I don't think it's too long, but lmk if you think it's better that they're separated out.
- [x] Include crowdin gem in Gemfile. 
- [x] Remove all the lists of languages and use cdo/languages instead.
- [x] Update the scripts to include some of the manual steps that I do inbetween.
- [x] Update sync-apps.sh to not have to include copy-untranslated-apps

**Todo** (not blocking merge):
- [ ] Update in.sh and out.sh scripts to ruby.
- [ ] Update the i18n/Rakefile so an engineer could get the API key for crowdin.yaml.
- [ ] Do the same updates for the Hour of Code and Curriculum scripts.
- [ ] Update [wiki](http://wiki.code.org/display/PROD/i18n)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,7 +18,6 @@
 /rebuild-apps
 /rebuild-shared
 /rebuild-code-studio
-/i18n/code.org/crowdin.yaml
```
Comment (mehalshah) [inline .gitignore:20]: If you remove this, will old enlistments stop ignoring i18n/code.org/crowdin.yaml? We might accidentally scoop it up - might as well ignore both locations

File: bin/i18n-codeorg/out.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```
Comment (mehalshah) [inline bin/i18n-codeorg/out.sh:1]: Give the script a verbose name / documentation, it's not clear to me what this does looking at it quickly

File: bin/i18n-codeorg/crowdin.yaml.erb [hunk 3]
```diff
@@ -21,4 +19,5 @@ files:
       - /source/blockly-mooc/applab.json
       - /source/blockly-mooc/calc.json
       - /source/blockly-mooc/eval.json
+      - /source/blockly-mooc/gamelab.json
```
Comment (tanyaparker) [inline bin/i18n-codeorg/crowdin.yaml.erb]: TODO: add to wiki how to specify which apps aren't translated. Needs to be updated in sync-codeorg and in crowdin.yaml.

File: bin/i18n-codeorg/in.sh [hunk 4]
```diff
@@ -19,13 +19,10 @@ git pull
```
Comment (tanyaparker) [inline bin/i18n-codeorg/in.sh]: TODO: add documentation explaining script

File: bin/i18n-codeorg/sync-pegasus.sh [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+	# Export to pegasus/cache/i18n/locales
+	orig_dir=pegasus/cache/i18n
+	loc_dir=i18n/locales/$locale/pegasus
+	en_dir=i18n/locales/source/pegasus
+
+	# Merge YML file.
+	ruby ./bin/i18n-codeorg/lib/merge-all-locales.rb "yml" $en_dir/mobile.yml $loc_dir/mobile.yml $orig_dir/$locale.yml
+	perl -i ./bin/i18n-codeorg/lib/fix-ruby-yml.pl $orig_dir/$locale.yml
+
+done
```
Comment (tanyaparker) [inline bin/i18n-codeorg/sync-pegasus.sh]: TODO: add documentation, fix indentation, add EOF line

**Pull request merged**